tests/test_training_simple.py

```diff
@@ -0,0 +1,16 @@
+from open_lm.main import main
+
+
+def test_train_simple():
+    seq_len = 2048
+    num_batches = 5
+    batch_size = 1
+    # fmt: off
+    main([
+        "--train-num-samples", str(num_batches * seq_len),
+        "--batch-size", str(batch_size),
+        "--dataset-type", "synthetic",
+        "--model", "open_lm_11m",
```

Comment:
maybe we should also add a "test config" which is even smaller? Like width is fine but maybe let's make it a 2-3 layer transformer? Call it `open_lm_test_config.json` or sth

Reply:
good idea, added